packages/app/src/components/swap/swap.tsx

```diff
@@ -53,6 +54,15 @@ const Swap: FC<Props> = ({ tokenSymbol, tokenBalance, tenderTokenBalance, protoc
 
   const sendInputRef = useRef<HTMLInputElement | null>(null);
 
+  const isSafeContext = isGnosisSafe()
+
+  const showApprove = (): boolean => {
```

Comment:
no need for explicit return type. Also this wouldn't have to be a function, but it's probably more readable this way